content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Announcing Istio 1.9
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9! Officially supported versions of Kubernetes releases are 1.17-1.20
+for this Istio release.
+
+{{< relnote >}}
+
+This is the first Istio release for 2021.  We would like to thank the entire Istio community, and especially the release
+managers Shamsher Ansari (RedHat), Steven Landlow (Google) and Jacob Delgado (Aspen Mesh) for helping to get Istio 1.9.0
+published.
+
+Given the work done by the UX Group, notably Mitch Connors (Google) and Christian Posta (Solo.io), and led by the
+direction of the [Istio TOC](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md), Istio 1.9's
+core focus was to improve Day 2 operations. These include stabilizing installation, upgrades, core functionality and
```

Comment:
Is there a glossary topic available for `Day 2 operations`? We should add some info or link about what is `Day 2 operations`.

Reply:
I added one. Please take a look at the definition. It needs some work to clarify what is meant.